gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.3-bin.zip
```

Comment:
the gradle update now generates a warning message. not sure how important it is. did you notice this?



  - Gradle detected a problem with the following location: 'G:\projects\osmdroid-caner\osmdroid-server-jdk\build\distributions\osmdroid-server-jdk-6.1.19-SNAPSHOT.zip'. Reason: Task ':osmdroid-server-jdk:publishPdZipPublicationPublicationToMavenLocal' uses this output of task ':osmdroid-server-jdk:distZip' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.6.3/userguide/validation_problems.html#implicit_dependency for more details about this problem.

> Task :osmdroid-server-jdk:publishReleasePublicationToMavenLocal
Multiple publications with coordinates 'org.osmdroid:osmdroid-server-jdk:6.1.19-SNAPSHOT' are published to repository 'mavenLocal'. The publications 'pdZipPublication' in project ':osmdroid-server-jdk' and 'release' in project ':osmdroid-server-jdk' will overwrite each other!